README.md

```diff
@@ -32,11 +32,13 @@ oc create -f deploy/converged/crds/hco.crd.yaml
 oc create -f deploy/converged/crds/kubevirt.crd.yaml
 oc create -f deploy/converged/crds/cdi.crd.yaml
 oc create -f deploy/converged/crds/cna.crd.yaml
+oc create -f deploy/converged/crds/kwebui.crd.yaml
 ```
 
 Launch all of the Service Accounts, Cluster Role(Binding)s, and Operators.
 ```bash
 oc create -f deploy/converged
+oc adm policy add-scc-to-user anyuid -z kubevirt-web-ui-operator
```

Comment:
this works when we are manually deploying these things but I was under the impression we were moving towards installing the HCO (and associated components) via OLM where anyuid is going to be a problem.

Reply:
https://github.com/kubevirt/web-ui-operator/pull/42

Reply:
I can look into this today.